syncer/streamer_controller.go

```diff
@@ -253,6 +256,12 @@ func (c *StreamerController) GetEvent(tctx *tcontext.Context) (event *replicatio
 	failpoint.Inject("GetEventError", func() {
 		err = errors.New("go-mysql returned an error")
 	})
+	failpoint.Inject("GetEventErrorOnce", func() {
+		if !mockGetEventErr {
```

Comment:
why we need check `mockGetEventErr` ?

Reply:
Otherwise, the same error occurs when we manually resume-task.

Reply:
oh i know, you don't want to remove failpoint and restart worker

Reply:
Can we do this by `"github.com/pingcap/dm/syncer/GetEventError=1*return"`?
Reference: https://github.com/pingcap/failpoint#key-concepts
Example: [dumpling failpoint](https://github.com/pingcap/dumpling/blob/843aff7/tests/basic/run.sh#L71)